utils/scraper.ts

```diff
@@ -117,29 +128,33 @@ export const scrapeKeywordFromGoogle = async (keyword:KeywordType, settings:Sett
       result: keyword.lastResult,
       error: true,
    };
+
    const scraperClient = getScraperClient(keyword, settings);
 
    if (!scraperClient) { return false; }
-   let res:any = null; let scraperError:any = null;
+
+   let res:any = null;
+   let scraperError:any = null;
+
    try {
       if (settings && settings.scraper_type === 'proxy' && settings.proxy) {
          res = await scraperClient;
       } else {
          res = await scraperClient.then((result:any) => result.json());
       }
 
-      if (res && (res.data || res.html || res.result || res.results)) {
-         const extracted = extractScrapedResult(res.data || res.html || res.result || res.results, settings.scraper_type);
+      if (res && (res.data || res.html || res.result || res.results || isSerpApiSearchResults(res))) {
+         const extracted = extractScrapedResult(res.data || res.html || res.result || res.results || res, settings.scraper_type);
```

Comment:
The last `|| res` is for SerpApi response. `isSerpApiSearchResults(res)` makes sure that `res` contains required properties.

Reply:
SerpApi returns organic results in `res.organic_results`. I've updated code to check and pass `res.organic_results` to the `extractScrapedResult`. And removed extra types that are not useful here.

Thanks for your feedback :+1: